osh/hub/scan/mock.py

```diff
@@ -116,23 +116,30 @@ def _get_build_arches(koji_proxy, task_id, tag):
     """
     returns all arches available for given build and tag
     """
+    def _get_tag_arches(tag):
+        # get all arches supported by given tag
+        # (which are stored in a string for some reason...)
+        return koji_proxy.getTag(tag)['arches'].split()
+
     arches = []
     for child_id in koji_proxy.getTaskDescendents(task_id, request=True):
         child = koji_proxy.getTaskInfo(child_id, request=True)
 
         # skip non-build subtasks
         method = child['method']
-        if method != 'buildArch':
+        if method not in ('buildArch', 'wrapperRPM'):
             continue
 
+        # wrapperRPM tasks do not track arches directly
+        if method == 'wrapperRPM':
```

Comment:
If this was inserted before the previous condition, the other condition could stay as it is.

Reply:
Good idea!

Reply:
Addressed in 2fe56b90291e32119ef69f8689b428c7424d1f44.